src/test/java/com/znsio/teswiz/screen/android/ajio/SearchScreenAndroid.java

```diff
@@ -33,10 +36,23 @@ public int numberOfProductFound() {
     }
 
     @Override
-    public void selectProduct() {
-        LOGGER.info("selection of Product in the result page");
-        driver.waitTillElementIsPresent(By.id("com.ril.ajio:id/layout_category_container")).click();
+    public ProductScreen selectProduct() {
+        LOGGER.info("Selection of Product in the result page");
+        if (!(driver.isElementPresent(byProductLayoutId))) {
```

Comment:
do we need extra braces in if condition? 
if(!driver.isElementPresent(byProductLayoutId))

Reply:
yes - when we land on the AJIO app on the first time - few sections got highlighted - and web elements are present below these elements